test/e2e/data/infrastructure-oci/v1beta1/cluster-template-managed-cluster-identity/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
+  - ./cluster.yaml
+  - ./machine-pool.yaml
+
```

Comment:
NIT: What is your thought on pulling the "identity" content out into it's own file, kind of like `cluster-template-cluster-identity/cluster-identity.yaml`? Only reason I suggest it is it makes it very clear there are identity changes, and keeps consistent with the other e2e test. 🤷

Reply:
fixed